torchmdnet/priors/__init__.py

```diff
@@ -1 +1,4 @@
 from torchmdnet.priors.atomref import Atomref
+from torchmdnet.priors.d2 import D2
+
+__all__ = ["Atomref", "D2"]
```

Comment:
why do we use __all__? I always found it sort of useless

Reply:
In this particular case, it gets used by the training script:

https://github.com/torchmd/torchmd-net/blob/9e0fef3f4f52ec6a40326d54cc71fd7675eb2ac9/torchmdnet/scripts/train.py#L66

It doesn't have to be implemented that way.  It just needs to get the list of allowed priors from somewhere.

Reply:
I see ok. Thanks!